packages/app/src/specs/generators/EmptyGenerator.vue

```diff
@@ -190,6 +195,25 @@ const hasError = computed(() => !isValidSpecFile.value && !!specFile.value)
 
 const result = ref<GeneratorSuccessFileFragment | null>(null)
 
+function getFileNameIndexes (inputValue: string, match: RegExpMatchArray): number[] {
+  return [match.index || 0, inputValue.indexOf(match[0]) + match[0].length]
+}
+
+onMounted(() => {
+  // Focus text input and try to pre-select file name
+  inputRef.value?.focus()
+
+  const fileNameRegex = /[ \w-]+(?=\.)/
```

Comment:
Can we get a comment here? My regex skills are sub-par so a comment would be helpful.

Reply:
I dig it!